src/sql/azdata.proposed.d.ts

```diff
@@ -517,6 +517,20 @@ declare module 'azdata' {
 		export function getNonDefaultOptions(profile: IConnectionProfile): Thenable<string>;
 	}
 
+	export interface ConnectionInfoSummary {
+		/**
+		 * Process ID returned from service host, if connection was successful.
```

Comment:
The "if connection was successful" is more of an implementation detail. Being a bit more generic here seems better if we want others to be able to use this.

```suggestion
		 * Process ID used to identify the connection on the server, if available.
```